SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -252,11 +262,25 @@ void USpatialGameInstance::OnLevelInitializedNetworkActors(ULevel* LoadedLevel,
 		return;
 	}
 
+	if (SpatialConnectionManager->IsConnected())
```

Comment:
I believe this will be null when running in native.

Reply:
Good catch, I'm not used to thinking of how things are different in native. Rather than put a condition here, I've changed the code to only register the delegate if we're running with SpatialOS, since otherwise offloading doesn't really make sense.

Reply:
That requires an engine PR as well then right?

Reply:
It shouldn't, I only register the callback now on line 206 if (HasSpatialNetDriver()).